conf/application.conf

```diff
@@ -250,3 +253,15 @@ hmrc-timeout-dialog {
   enableSynchroniseTabs = true
   defaultCountdownInSeconds = 120
 }
```

Comment:
The property "contact-frontend.serviceId" looks to be missing. This allows the link "Is this page not working properly? (opens in new tab)" to be displayed